towncrier.toml

```diff
@@ -1,37 +1,68 @@
 [tool.towncrier]
+name = "Jobbergate"
 directory = "changes"
 filename = "CHANGELOG.md"
 start_string = "<!-- towncrier release notes start -->\n"
-title_format = "## [{version}](https://github.com/omnivector-solutions/jobbergate/releases/tag/{version}) - {project_date}"
-issue_format = "[ASP-{issue}](https://jiratoolchain.collaborationlayer-traton.com/browse/ASP-{issue})"
+title_format = "# [{version}](https://github.com/omnivector-solutions/jobbergate/releases/tag/{version}) - {project_date}"
+issue_format = "[PR #{issue}](https://github.com/tomnivector-solutions/jobbergate/pull/{issue})"
```

Comment:
I like this a lot better, because the associated ticket number will be found in the PR (if there is one). Nice.